docs/dev-notes/2026-01-02/migrate-from-svelte-5-ui-lib-to-flowbite-svelte/README-plan.md

```
@@ -0,0 +1,485 @@
+    - [ ] mobile (375px) での動作 ⏭️ SKIP（`test.fixme()`でマーク）
+      - 原因: Flowbite-Svelte v1.31.0 の navbar `breakpoint` prop が Svelte 5 Context API 実装の不具合で機能しない
+      - 対応: v2.0 リリース待機推奨
+      - GitHub Issue: https://github.com/themesberg/flowbite-svelte/issues/1710
+
+- [ ] logout テスト失敗の調査・修正
+  - [ ] 原因: navbar header dropdown の実装・状態確認
+  - [ ] アクション: Header.svelte の dropdown 実装を確認し、ユーザー名リンクの可視性をテスト
+  - [ ] 期待時期: フェーズ1-4 完了後
+
+- [ ] visual regression テスト（手動確認）
+  - [ ] space-y-_ / space-x-_ セレクタ変更による layout shift 確認
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)